### PR TITLE
Minutes/20230110

### PR DESCRIPTION
**Pull request by florcabral**
Add minutes for January 2023 meeting

### CODE REVIEW

File: meetings/2022_04_12/README.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# ROS 2 Security Working Group Meeting Minutes
+ 
+12 April 2022
+[Meeting Recording](https://www.youtube.com/watch?v=C_3vnW8rQPA) | [Meeting Announcement](https://discourse.ros.org/t/security-working-group-meeting-march-2022/24528)
```
Comment (ruffsl) [inline meetings/2022_04_12/README.md]: These URLs seem to link to the same Meeting Recording and Announcement as from 08 March 2022. Could this be corrected? Thanks!

**Review by ruffsl — CHANGES_REQUESTED** [review]

File: meetings/2022_04_12/README.md (hunk 1 above)
Comment (florcabral) [reply meetings/2022_04_12/README.md]: The meeting announcement link is updated.

Comment (ruffsl) [reply meetings/2022_04_12/README.md]: What about the YouTube link though? The same YouTube URL code wouldn't be used twice, no?

Comment (artivis) [reply meetings/2022_04_12/README.md]: This is now fixed. Thanks for the catch :+1:

**Review by artivis — APPROVED** [review]

**Review by ruffsl — APPROVED** [review]

**Pull request merged**